MAUI/Maps/legend.md

```diff
@@ -660,4 +660,172 @@ public class Model
 
 ![Legend position](images/legend/legend-position.png)
 
+## Appearance customization
+
+You can customize the legend items using the following properties.
```

Comment:
Customize the legend items using the following properties.

Reply:
I have changed.